### PR TITLE
fix(ci): Increase timeout for odsp and frs perf benchmarks

### PR DESCRIPTION
**Pull request by alexvy86**
## Description

After the recent introduction of contention locks for the odsp and frs stages of the Performance Benchmarks pipeline (#15294), those stages started timing out / getting canceled if they couldn't obtain the lock within the default 60 min timeout for a stage. This change increases the timeout for those two stages to 360 minutes, to match the e2e tests pipeline.

Confirmed with a run from a test/ branch with these changes that the new timeout is being applied:

image

It also adds the `lockBehavior: sequential` configuration so that every run of the pipeline executes those stages as the lock becomes available, instead of the latest run executing them and previous runs skipping them (if there are several runs queued), which is the default behavior when using these locks.

### CODE REVIEW

**Comment by alexvy86** [comment]
The Performance benchmarks pipeline is not a requirement for PR, it shows up here because of my manual run to test the change, and the failures are for a different reason. I'll proceed with merging this once it is approved as long as all other checks pass.

**Review by tylerbutler — APPROVED** [review]

**Pull request merged**
